ramin/src/index.ts

```diff
@@ -0,0 +1,58 @@
+//Ts Basic
+
+// variable : 
+const personName:string = "ramin";
```

Comment:
The self "TypeScript" infers the types
```suggestion
const personName = "ramin";
```